src/lineup/ALineUpView2.ts

```diff
@@ -444,7 +444,14 @@ export abstract class ALineUpView2 extends AView {
             colDesc.domain[1] = d3.max(values, (d) => d.max);
           }
         }
-        col.setLoaded(true);
+
+        // find all columns with the same descriptions (generated snapshots) to set their `setLoaded` value
+        const rankings = this.lineup.data.getRankings();
```

Comment:
since a couple of lines later it will be checked if lineup exists, you should do the same